Documentation/docs/using-fluent.md

```diff
@@ -0,0 +1,220 @@
+
+# Developer Guide : Using Fluent theme in WPF in .NET 9
+
+In .NET 9, as part of the ongoing modernization in WPF, we introduced the Fluent ( Windows 11 ) theme. However there are still a lot of gaps that need to be filled. In this document, we describe the behaviors and general ways of using the new Fluent theme. Key-enhancements include : 
+
+- Enable Fluent theme at two levels : Application and Window
+- Support for Light and Dark themes
+- Accent color support
+- Default backdrop for windows when using Fluent theme
+
+## Using Fluent theme in your WPF applications
+
+Enabling Fluent theme in WPF application is supported at two levels : **Application** and **Window**.
+
+There are two ways in which you can use \ enable Fluent theme in your WPF Applications - including the Fluent theme resource dictionaries or you can use the experimental **ThemeMode** APIs.
```

Comment:
It should be noted here that they are the same in effect, i.e. the API only includes the resource dictionary as convenience.